barretenberg/cpp/src/barretenberg/vm2/common/protocol_contract_data.cpp

```diff
@@ -5,7 +5,7 @@ namespace bb::avm2 {
 
 const std::unordered_map<CanonicalAddress, DerivedAddress> derived_addresses = {
     { CANONICAL_AUTH_REGISTRY_ADDRESS, AztecAddress("0x1c2474a77af3c756895867fa7622e5e9295121b35849e965f4614d51f00c62e2") },
-	{ CONTRACT_INSTANCE_REGISTRY_CONTRACT_ADDRESS, AztecAddress("0x001e627e78063a4944f1331b4c59544f8ad6d6d478f2fc79b025e8240b94b816") },
+	{ CONTRACT_INSTANCE_REGISTRY_CONTRACT_ADDRESS, AztecAddress("0x1f4284dea3e475d9d40283c7fe4c1aef5cdea59c012a8fa84c0c9ebe0752327e") },
```

Comment:
For context see https://aztecprotocol.slack.com/archives/C02M7VC7TN0/p1757341750525609